e2e/pages/view-hearing.po.ts

```diff
@@ -17,4 +19,20 @@ export class ViewHearingPage {
     getHeaderText() {
         return this.header.getText();
     }
+
+    getStatusText() {
+        return this.status.getText();
+    }
+
+    async chooseAnActionFromDropDown(optionName: string) {
```

Comment:
we have sth like this already in repo ^^ 
Take a look at *selectValueFromSingleSelectOption* which is in *ElementHelper*